### PR TITLE
Adjust documentation to stay up-to-date with the removal of `cost`

### PR DESCRIPTION
**Pull request by fdtaillefer**

### CODE REVIEW

File: logicalRequirements.md [hunk 1]
```diff
@@ -79,43 +72,206 @@ An `enemyKill` object can have the following properties:
...
+
+#### heatFrames object
+A `heatFrames` object represents the need for Samus to spend time (measured in frames) in a heated room. This is meant to be converted to a flat health value based on item loadout. The vanilla damage for heated rooms is 1 damage every 4 frames, negated by Varia or Gravity Suit.
+
+__Example:__
+```json
+{"heatFrames": 100}
+```
+
+#### hibashiHits object
```
Comment (DIProgan) [inline logicalRequirements.md]: Nice picking this up. Now the metroidFrames ^^

Comment (fdtaillefer) [reply logicalRequirements.md]: That probably shouldn't be a logical element. It's an attack performed by Metroids. The randomizer suits reduce it like enemy damage, too.

Comment (fdtaillefer) [reply logicalRequirements.md]: Draygon electricity frames are less clear. I'll take a finer look and possibly add them in the doc.

**Comment by DIProgan** [comment]
Merge away

**Pull request merged**